service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FileUploadNotificationReceiver.java

```diff
@@ -211,14 +210,8 @@ public FileUploadNotification receive(long timeoutMs) throws IOException, Interr
     {
         final CompletableFuture<Void> future = new CompletableFuture<>();
         executor.submit(() -> {
-            try
-            {
-                close();
-                future.complete(null);
-            } catch (IOException e)
-            {
-                future.completeExceptionally(e);
```

Comment:
What is the impact of removing this?

Reply:
Nothing. Since the IOException was never thrown here, the catch block would never actually execute